.github/workflows/release.yml

```diff
@@ -9,31 +9,46 @@ name: Create Release
 jobs:
   build:
     name: Create Release
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
-      - uses: little-core-labs/get-git-tag@v3.0.1 # https://github.com/olegtarasov/get-tag/issues/14
-        id: tagName
+        uses: actions/checkout@v3
         with:
-          tagRegex: "v(.*)"
-          tagRegexGroup: 1
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+          path: snapshot-streaming
+
+      - name: Checkout tessellation
+        uses: actions/checkout@v3
+        with:
+          path: tessellation
+          repository: Constellation-Labs/tessellation
+          ref: v1.9.1
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
-      - name: Assemble assets
-        run: |
-          sbt "++ 2.12.10" assembly
+          distribution: temurin
+          java-version: 11
```

Comment:
Do we want to stick with java 11 rather than 8?

Reply:
Well, I don't think we should stick with any Java version. I think we should just advance it to the newer LTS versions that are compatible with our software whenever we find it suitable (jdk 1.8 was released almost a decade ago). I know that testnet and mainnet instances of snapshot-streaming are currently running jdk 1.8, but I think we can update them to java 11 as well.